index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Document</title>
+</head>
+<body>
+   
+   <!-- header -->
+   <header>header</header>
+   
+   <!-- feature a -->
+   <div class="feature-a">feature A</div>
+
+   
+   <!-- feature a -->
+   <div class="feature-a">deature B</div>
```

Comment:
this should be `class="feature-b`

Reply:
Thanks @WarriorofZarona (just updated it) (please review again)